.travis.yml

```diff
@@ -29,4 +29,5 @@ script:
   - doxygen doxygen/doxygen.conf
```

Comment:
can we replace `addrbook.c body.c complete.c compress.c [...]`  with `./*.c` ? 

Might have nothing to do with your PR, but maybe we could fix that here.

Reply:
This list will become `*.c` in the future.
For now, it's the files that I've tidied.